src/useEyeDrop.ts

```diff
@@ -54,6 +55,14 @@ export const useEyeDrop = ({
     const targetCanvas = await targetToCanvas(target)
     const rgbColor = getColor(targetCanvas, e, pickRadius)
 
+    if (onChange) {
+      const rgb = parseRGB(rgbColor)
+      const hex = rgbToHex(rgbColor)
+
+      // set color object to parent handler
+      onChange({ rgb, hex, customProps: undefined })
```

Comment:
If we do decide to go with onChange instead of returned value, let's add the customProps in the call? 🤠

Reply:
I couldn't find what customProps was at the time.  But sure, I'll add it.